gpustat/core.py

```diff
@@ -379,6 +379,8 @@ def print_formatted(self, fp=sys.stdout, force_color=False, no_color=False,
                         show_cmd=False, show_user=False, show_pid=False,
                         show_power=None, gpuname_width=16,
                         show_header=True,
+                        eol_char='\n',
```

Comment:
Is this eol character portable for OS other than POSIX? I think windows use '\n\r' or '\r\n'. However, it doesn't matter if we target on only POSIX systems.

Reply:
`os.linesep` can be a better choice of default parameter.

Reply:
Yeah, it was previously just '\n'. I would say POSIX systems are just targetted, but totally fine with using `os.linesep`.